libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/thrift/ThriftValidate.java

```diff
@@ -244,6 +245,28 @@ public static void validateAttachment(AttachmentContent attachment) throws SW360
         attachment.setType(TYPE_ATTACHMENT);
     }
 
+    public static void validateLicenseIdMatch(License license) throws SW360Exception {
+        if (license.isSetId() && license.isSetShortname()) {
+            String message = "license short name must be equal with license id";
```

Comment:
I believe "equal to" is the grammatically correct phrase

Reply:
corrected